settings.gradle

```diff
@@ -1,3 +1,3 @@
 include 'wallet'
 include 'integration-android'
-include 'sample-integration-android'
```

Comment:
This looks like it was done to remove annoyances from this example, but it has nothing to do with wall of coins support.